client/document/utils/index.js

```diff
@@ -0,0 +1,2 @@
+export { chunkCssLinks } from './chunk';
```

Comment:
Note: I added this file to avoid having to update all the other modules importing `./utils` and keep the changes in this PR a bit more focused. I'd be happy to remove it and update all the other modules instead, though, if you think that's best.